EpamWebSiteTestingFramework/Epam.TestAutomation.Pages/PageObjects/Pages/JobDetailsPage.cs

```diff
@@ -0,0 +1,66 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Core.Utils;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Pages.PageObjects.Pages;
+
+public class JobDetailsPage : BasePage
+{
+    private JobSearchResultPage _resultPage;
```

Comment:
why do you need links from one page to another? 
it breaks all code structure. Cases when you need to open 1 page and then do something on another better implement in tests or steps